### ISSUE TITLE
Babel version too old

### ISSUE THREAD

**Issue by hajdun** [lukejpreston/xunit-viewer#50]
I cannot use xunit viewer (5.1.10)
```
node_modules/.bin/xunit-viewer --results=junit.xml --output=file_location.html
```
, because it has 6.23.... babel core and I get this message

```
Error: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel. while parsing file: .../node_modules/clone/clone.js
    at component.then.catch.err (.../node_modules/xunit-viewer/cli/render.js:33:20)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Done in 2.41s.

```

**Comment by hajdun** [comment]
updates suggested:
package json
```
{
  "name": "xunit-viewer",
  "version": "5.1.10",
  "description": "parses xunit xml into xunit viewer",
  "repository": {
    "url": "https://github.com/lukejpreston/xunit-viewer.git",
    "type": "git"
  },
  "bugs": {
    "url": "https://github.com/lukejpreston/xunit-viewer.git"
  },
  "homepage": "https://lukejpreston.github.io/xunit-viewer",
  "keywords": [
    "test",
    "junit",
    "xunit",
    "viewer",
    "parser"
  ],
  "author": "lukejpreston <lukejpreston@gmail.com>",
  "license": "MIT",
  "main": "xunit-viewer",
  "bin": {
    "xunit-viewer": "bin/xunit-viewer"
  },
  "directories": {
    "bin": "bin",
    "cli": "cli",
    "component": "component",
    "config": "config",
    "parser": "parser"
  },
  "scripts": {
    "test": "npm run eslint && jest",
    "jest": "jest",
    "eslint": "eslint bin cli component config data parser scripts index.js && echo eslint pass",
    "eslintfix": "eslint bin cli component config data parser scripts index.js --fix && echo eslint pass",
    "start": "nodemon --watch component --exec npm run run",
    "run": "./bin/xunit-viewer --results=data  --ignore=invalid.xml,blank_file.xml --output=output.html",
    "xunit-viewer": "./bin/xunit-viewer",
    "xunit-viewer:sockets": "./bin/xunit-viewer --results=data --dev --watch  --ignore=invalid.xml,blank_file.xml --port=8080",
    "xunit-viewer:terminal": "./bin/xunit-viewer --results=data  --ignore=invalid.xml,blank_file.xml --terminal --watch --filter.suites.search=\"bacon\" --filter.suites.hidden=pass,fail --filter.tests.search=\"bacon\" --filter.tests.hidden=error,skip --filter.properties.search=\"x\" --filter.properties.hidden=all ",
    "xunit-viewer:output": "./bin/xunit-viewer --results=data  --ignore=invalid.xml,blank_file.xml --watch --output=output.html"
  },
  "dependencies": {
    "@babel/core": "^7.4.5",
    "babel-preset-react-app": "^2.1.1",
    "babelify": "^7.3.0",
    "browserify": "^16.2.3",
    "bulma": "^0.3.2",
    "chalk": "^1.1.3",
    "core-js": "^3.2.1",
    "detect-port": "^1.1.0",
    "express": "^4.15.2",
    "fs-extra": "^2.0.0",
    "fuzzysearch": "^1.0.3",
    "mustache": "^2.3.0",
    "node-watch": "^0.4.1",
    "normalize.css": "^5.0.0",
    "postcss": "^5.2.15",
    "postcss-cssnext": "^2.9.0",
    "postcss-preset-env": "^6.7.0",
    "promise": "^7.1.1",
    "react": "^15.4.2",
    "react-dom": "^15.4.2",
    "recursive-readdir": "^2.1.1",
    "socket.io": "^2.1.1",
    "titlecase": "^1.1.2",
    "uglify-js": "^2.8.7",
    "uglifycss": "^0.0.25",
    "uuid": "^3.0.1",
    "xml2js-parser": "^1.1.1"
  },
  "devDependencies": {
    "eslint": "^3.17.0",
    "eslint-config-standard": "^7.0.0",
    "eslint-config-standard-react": "^4.2.0",
    "eslint-plugin-jasmine": "^2.2.0",
    "eslint-plugin-promise": "^3.5.0",
    "eslint-plugin-react": "^6.10.0",
    "eslint-plugin-standard": "^2.1.1",
    "jest": "^24.3.1",
    "nodemon": "^1.18.7"
  },
  "jest": {
    "testPathIgnorePatterns": [
      "<rootDir>[/\\\\](node_modules|scripts)[/\\\\]",
      "<rootDir>/component/suites/test.jsx",
      "<rootDir>/component/unbundled.js"
    ]
  }
}
```

**Comment by lukejpreston** [comment]
Could you try upgrading to `5.1.11`?

I have updated as many dependencies as possible.

**Comment by lukejpreston** [comment]
I am going to close this issue, if you see it crop up again using that version let me know :D

**Comment by cbowal** [comment]
@lukejpreston you haven't released 5.1.11 to npm

**Comment by lukejpreston** [comment]
Sorry for the delay. I have published it now
